src/routes.ts

```diff
@@ -0,0 +1,51 @@
+import express, { Express } from "express";
+import axios from "axios";
+import { API_KEY } from "./info";
+import { checkCameraType } from "./cameras";
+import { reduceInfo } from "./photo";
+import { getHtml } from "./helper";
+
+export const router = express.Router();
+
+// Part 2
+
+router.get("/rovers", function (req, res) {
+
+    console.log("Getting rovers ...");
```

Comment:
Except for a few cases (starting the server, logging errors), we prefer to have as few `console.log`s as possible. They're very useful when debugging and writing your code, but we don't want them to be in code we run in production